optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/result/BenchmarkResultIO.java

```diff
@@ -108,9 +99,6 @@ protected PlannerBenchmarkResult readPlannerBenchmarkResult(File plannerBenchmar
             String benchmarkReportDirectoryName = plannerBenchmarkResultFile.getParentFile().getName();
             plannerBenchmarkResult = PlannerBenchmarkResult.createUnmarshallingFailedResult(
                     benchmarkReportDirectoryName);
-        } catch (XStreamException e) {
```

Comment:
IDE was reporting this as having no effect, since ```XStreamException extends IOException```. That said, if you want it there, please put it back when you're merging.